rmw_implementation-extras.cmake.in

```diff
@@ -16,18 +16,8 @@
 
 find_package(rmw_implementation_cmake REQUIRED)
 
-if(NOT "${RMW_IMPLEMENTATION}" STREQUAL "")
-  set(requested_rmw_implementation "${RMW_IMPLEMENTATION}")
-  set(requested_rmw_implementation_from "CMake")
-elseif(NOT "$ENV{RMW_IMPLEMENTATION}" STREQUAL "")
-  set(requested_rmw_implementation "$ENV{RMW_IMPLEMENTATION}")
-  set(requested_rmw_implementation_from
-    "environment variable 'RMW_IMPLEMENTATION'")
-else()
-  set(requested_rmw_implementation "@RMW_IMPLEMENTATION@")
-  set(requested_rmw_implementation_from
-    "the default when @PROJECT_NAME@ was built")
-endif()
+get_default_rmw_implementation(requested_rmw_implementation)
```

Comment:
We might want to warn about the fact that we are not honoring the `RMW_IMPLEMENTATION` variable, if I understand the changes correctly.

Reply:
It's been a while since I originally made this change, but IIRC, the `get_default_rmw_implementation` does honor `RMW_IMPLEMENTATION`. The only difference in behavior is when `RMW_IMPLEMENTATION` isn't specified and the default RMW isn't available, it falls back on any available RMW instead of falling back to a specific hard-coded RMW.